### PR TITLE
Ben's AD review comments

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

File: draft-ietf-git-using-github.md [hunk 1]
```diff
@@ -72,8 +72,10 @@ A companion document, {{?GH-CONFIG=I-D.ietf-git-github-wg-configuration}},
 describes administrative processes that support the practices described in this
 document.
 
-Although similar, guidance for IRTF Research Groups is out of scope for this document.
-However, such groups may draw inspiration for GitHub use from the contents herein.
+Although the operation of IRTF Research Groups can be similar in function to
+Working Groups, this document only directly addresses the needs Working Groups.
```
Comment (kaduk) [inline draft-ietf-git-using-github.md]:
```suggestion
Working Groups, this document only directly addresses the needs of Working Groups.
```

File: draft-ietf-git-using-github.md [hunk 2]
```diff
@@ -168,17 +171,16 @@ found.  For instance, the name could be ietf-wg-\<wgname>, as recommended in
...
-owners.
+GitHub requires that each organization have at least one owner.  The owners for
+a Working Group repository MUST include responsible Area Directors.  Area
+Directors MAY also designate a delegate that becomes an owner and Working Group
+chairs MAY also be owners.
 
-A team with administrator access SHOULD be created for the Working Group Chairs
-and any Working Group Secretary.  Administrator access is preferable, since this
-does not also include the ability to push to all repositories and ownership does
-not grant any other significant privileges.
```
Comment (kaduk) [inline draft-ietf-git-using-github.md:192]: I can understand that there's not a huge need to mention the details of Administrator privilege in these guidelines.

File: draft-ietf-git-using-github.md [hunk 3]
```diff
@@ -512,14 +514,15 @@ formal standing in the Working Group process.
...
+for tracking outstanding issues.  In this mode of interaction, a record of the
+existence of substantive technical discussions is tracked using issues in the
+issue tracker.  However, discussion of any substantial matters is always
+conducted on mailing lists.
 
 Under this mode, issues and pull requests can be opened by anyone, but anything
 deemed substantive MUST be resolved exclusively on the mailing list.  Discussion on
-GitHub is kept to a minimum.  Only editorial matters can be resolved using the
-issue tracker.
+GitHub is kept to a recording the state of issues.  Only editorial matters can
```
Comment (kaduk) [inline draft-ietf-git-using-github.md]:
```suggestion
GitHub is limited to recording the state of issues.  Only editorial matters can
```

File: draft-ietf-git-using-github.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-git-using-github.md:192]: Well, the point was that this was flat wrong, based on an outdated authorization model.

File: draft-ietf-git-using-github.md [hunk 4]
```diff
@@ -173,10 +173,10 @@ A single organization SHOULD NOT be used for all IETF activity, or all activity
 within an area. Large organizations create too much overhead for general
 management tasks.
 
-Each organization requires owners.  The owner team for a Working Group
-repository MUST include responsible Area Directors.  Area Directors MAY also
-designate a delegate that becomes an owner and Working Group chairs MAY also be
-owners.
+GitHub requires that each organization have at least one owner.  The owners for
+a Working Group repository MUST include responsible Area Directors.  Area
+Directors MAY also designate a delegate that becomes an owner and Working Group
```
Comment (chris-wood) [inline draft-ietf-git-using-github.md]:
```suggestion
Directors MAY also designate a delegate that becomes an owner. Working Group
```

File: draft-ietf-git-using-github.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-git-using-github.md:192]: Should we mention that the model (as is now documented) is also subject to change?

File: draft-ietf-git-using-github.md [hunk 5]
```diff
@@ -195,7 +195,8 @@ organization.
 
 A simple example of how to do this is to include a link to the GitHub
 organization on the WG Charter page in the datatracker.  Similarly, if there are
-multiple mailing list options, links to those mailing lists should be given.
+additional resources, such as additional mailing lists, links to those resources
```
Comment (chris-wood) [inline draft-ietf-git-using-github.md]:
```suggestion
additional resources, such as mailing lists, links to those resources
```

**Review by chris-wood — APPROVED** [review]

**Pull request merged**